src/errors/ApiError.ts

```diff
@@ -0,0 +1,24 @@
+class ApiError extends Error {
+  status: number;
+  message: string;
+  constructor(status: number, message: string) {
+    super();
+    this.status = status;
+    this.message = message;
+  }
+
+  static badRequest(message: string) {
```

Comment:
fabrik methods usually starts with a word 'make' or 'create'

all methods should be named with verbs